packages/url/src/is-email.js

```diff
@@ -1,4 +1,4 @@
-const EMAIL_REGEXP = /^(mailto:)?[a-z0-9._%+-]+@[a-z0-9][a-z0-9.-]*\.[a-z]{2,63}$/i;
+const EMAIL_REGEXP = /^(mailto:)?[a-z0-9._%+-]+@[a-z0-9][a-z0-9.-]*\.[a-z0-9]{2,63}$/i;
```

Comment:
This will fail for IPs with a single number at the end `ace@1.1.1.1` and succeed for incorrect IPs, like one with a number higher than 255 (`ace@900.1.1.1`), with 4 numbers (`ace@1111.1.1.1`), or with less or more than 4 sets of numbers (`ace@1.1.1`, `ace@1.1.1.1.1`).

Why don't we add a proper IP regex, like `^((1?[1-9]?\d|25[0-5]|2[0-4]\d)\.){3}(1?[1-9]?\d|25[0-5]|2[0-4]\d)$`? 🙂